content/en/synthetics/guide/api_test_timing_variations.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Understanding API test timings and troubleshooting their variations
```

Comment:
How does "API Test Timing Metrics and Variations" sound as a page title?

```suggestion
title: API Test Timing Metrics and Variations
```

Reply:
Also, how does this page relate to the Synthetic Metrics page (https://docs.datadoghq.com/synthetics/metrics/#metrics)?

Reply:
IMO, having this page separate will allow us to keep the Synthetics Metrics page short and concise, like a metric 'inventory'. That said, linking the guide in the metrics page will give users possibility to learn more about metrics in the guide.

What do you think @MargotLepizzera

Reply:
Yeah this one is definitely tricky. 
The first section of the doc would definitely make sense to include in the existing metrics page but it felt like it was too much info to fit in the metrics table @alai97. Plus I believe what's in the table is also used in the `Metrics summary` page and I feel like it would be weird to have so much content in the `Metrics summary`?
The second part however is really about troubleshooting variations so I'd rather keep it separate. 
I think we can go with this for now, and maybe add a card in the docs backlog to think about how to better integrate the first section in the existing metrics page?

Reply:
Gotcha, that rationale sounds good to me!